src/utils.js

```diff
@@ -205,20 +205,27 @@ const Utils = module.exports = {
 
   /**
    * Returns a function that will throw when called. Useful to ensure certain methods are overridden.
-   * 
+   *
    * @method missingImplementation
    * @static
    * @param {String} name
    */
   missingImplementation(name) {
     throw new Error(`${this && this.constructor && this.constructor.name} must implement ${name}()`);
   },
-  
-  
+
+
   makeArray(value) {
     if (Array.isArray(value)) return value;
     if (_.isUndefined(value)) return [];
     return [value];
+  },
+
+  addIfNotInclude(array, item) {
+    if (!_.includes(array, item)) {
```

Comment:
Keeping lodash because native contains/includes breaks in different node versions